docs: Revamp manage-tokens chapters (#777)
oasisprotocol/docs: docs/general/manage-tokens/holding-rose-tokens/ledger-wallet.md

## Setup your Ledger device and Install Oasis App

1. To use your [Ledger] wallet to hold your ROSE tokens, you will have to
   install [Oasis app] on your Ledger wallet via [Ledger Live]'s Manager. Click
   on the "My Ledger" button. Then, you need to connect your Ledger to your
   computer and unlock it with your PIN code.

   ![Unlock ledger](../../images/wallet/ledger/live_unlock_ledger.png)

   :::caution

   The Oasis app requires an up-to-date firmware on your Ledger wallet:

>    * At least [version 2.0.0] released on Oct 21, 2021 on a Nano X device.

Reviewer: (out of scope) I wonder if we should bump these.

Author: No, we don't bump those in the markdown sources, because it just eats too much space.

   * At least [version 1.0.4] released on Sep 27, 2022 on a Nano S Plus device.

   Follow Ledger's instructions for updating the firmware on your Ledger wallet:

   * [Nano X]
   * [Nano S]
   * [Nano S Plus]

   :::

2. Next, allow the Ledger Manager on your Ledger device. Then you will be able
   to open the App catalog and search for `oasis`:

   ![Allow Ledger Manager](../../images/wallet/ledger/live_allow_ledger_manager.png)
   ![Search app in catalog..](../../images/wallet/ledger/live_search_apps.png)

3. Install the **Oasis** Nano app.

   ![Install the Oasis Nano app](../../images/wallet/ledger/live_search_results_oasis_install.png)

4. After the installation is complete, take your Ledger device, navigate to the
   Oasis app and use both buttons to open it. Your Ledger device is ready when
   you will see the "Oasis Ready" message.

   ![Oasis Ready](../../images/wallet/ledger/ledger_oasis_ready.jpg)

The Oasis app will use the [BIP 39] mnemonic seed stored secretly on your
Ledger hardware wallet to generate the private & public key pairs for your
Oasis accounts. **Make sure you backed up the mnemonic when you first
initialized your Ledger device!**

:::tip

> The Ledger device uses a different derivation path from the standardized [ADR-8]

Reviewer 2: Hm what? The Ledger device supports both legacy and adr8 derivation paths.

Author: Added "by default".

used by the Oasis wallets. If you find yourself in a situation where you Ledger
device does not function anymore, you have the backup mnemonic available and you
urgently need to access your funds, you can use the *Oasis unmnemonic tool* to **derive the private key
from your Ledger mnemonic**. Checkout out this [FAQ section][unmnemonic-tool] to
learn more.

:::

[ADR-8]: ../../../adrs/0008-standard-account-key-generation.md
[unmnemonic-tool]: ../faq.mdx#i-lost-my-ledger-or-my-ledger-is-broken-i-urgently-need-to-access-my-assets-can-i-import-ledger-mnemonic-into-oasis-wallet

## Connect to your wallet

### Oasis Wallets

This is a simpler option since it offers a nice UI for connecting your Ledger to
a web application or a browser extension. Check out the following sections
corresponding to your wallet for instructions:

- [Oasis wallet - Web: Import Ledger account](../oasis-wallets/web.mdx#import-an-existing-account)
- [Oasis wallet - Browser extension: Ledger](../oasis-wallets/browser-extension.mdx#ledger)

:::note

> At time of writing, signing the ParaTime transactions was not yet supported by

Reviewer: ```suggestion
At time of writing, signing ParaTime transactions is not yet supported by
```
